### DISCUSSION TITLE
Question about gist

### DISCUSSION THREAD

**Discussion by coccococco06** [community/community#39170] [New to GitHub] [answered]
Hi, i want my code embedded in another page, but i want it to be in dark theme and not white, what can i do?

**Comment by david-engelmann** [comment, marked as answer]
[You can modify the gist theming with css, here's a great guide](https://danielbernal.co/styling-gist-embeds-with-css/). [I'll also include a sample css file with dark mode setup](https://gist.github.com/just-jeb/27d97d675c7d21ed441dd1458f72cb4a)!

**Reply by coccococco06** [reply]
i forgot to thank you, sorry!
